README.md

```diff
@@ -35,7 +35,7 @@
 
 ## Overview
 <p align="justify">
-DMeta is an open source Python package that provides ... TODO
+DMeta is an open source Python package that removes metadata of microsoft office files.
```

Comment:
I will change it to `... Microsoft Office ...`, is that OK?

Reply:
Sure, go ahead.